nuget/framework/nunit.nuspec

```diff
@@ -50,7 +49,11 @@ Supported platforms:
     <file src="bin/net-4.5/nunit.framework.xml" target="lib/net45" />
     <file src="bin/netstandard16/nunit.framework.dll" target="lib/netstandard1.6" />
     <file src="bin/netstandard16/nunit.framework.xml" target="lib/netstandard1.6" />
-    <file src="bin/portable/nunit.framework.dll" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac+MonoAndroid10+MonoTouch10+Xamarin.iOS10" />
-    <file src="bin/portable/nunit.framework.xml" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac+MonoAndroid10+MonoTouch10+Xamarin.iOS10" />
+    <file src="bin/portable/nunit.framework.dll" target="lib/portable-net45+win8+wp8+wpa81" />
+    <file src="bin/portable/nunit.framework.xml" target="lib/portable-net45+win8+wp8+wpa81" />
+    <file src="bin/portable/nunit.framework.dll" target="lib/MonoAndroid" />
+    <file src="bin/portable/nunit.framework.xml" target="lib/MonoAndroid" />
+    <file src="bin/portable/nunit.framework.dll" target="lib/Xamarin.iOS10" />
+    <file src="bin/portable/nunit.framework.xml" target="lib/Xamarin.iOS10" />
```

Comment:
I presume we're assuming here that nobody actually uses the Xamarin.Mac/MonoTouch targets? 😄 I wonder how far away we are from netstandard 1.4, which (I think?) would just be consumable by all Xamarin targets...